### ISSUE TITLE
Accessibility: Multiple h1 elements

### ISSUE THREAD

**Issue by sedlatschek** [laravel/nova-issues#3429]
- Laravel Version: 8.44.0
- Nova Version: 3.26.1
- PHP Version: 7.4.6
- Database Driver & Version: MySQL 5.7
- Operating System and Version: Win10
- Browser type and version: Chrome 88

### Description:
Having more than one `h1` heading makes it hard for blind users to grasp the page contents. There should be only one `h1` which then describes the contents of the current page. 

Currently every panel has an `h1` heading. Please use `h2+` instead.

![image](https://user-images.githubusercontent.com/58665536/121051704-f5b3a380-c7b9-11eb-8b7e-1180d538e5ab.png)

Laravel Nova is not accessible. This is a problem in enterprise environments, where accessibility is a must requirement.

**Comment by davidhemphill** [comment]
HTML5 permitted multiple H1 elements because of the semantics provided by the element types added in the spec. It would be most helpful for the requests to provide a link to a reputable source for the information offering the alternatives.

It would also be appreciated to discontinue adding "Laravel Nova is not accessible. This is a problem in enterprise environments, where accessibility is a must requirement." to every PR. You have overstated your point. :-)

**Comment by davidhemphill** [comment]
👋 Hi there! In order to keep this repository focused on bug reports, we auto-close feature requests and requests for help. Feel free to post your feature requests so others can discuss and add reactions. We'll keep an eye on them for later planning.

**Comment by davidhemphill** [comment]
Hey, I appreciate you bringing this to our attention. It is something we care about, but is admittedly pretty difficult to find reputable information that doesn't conflict with other reputable sources. 😅 

We will be making a big accessibility sweep across the product in the coming months in preparation for the next release. We will likely bring those changes to the 3.x branch in Orion where it makes sense.
